de-lib/src/main/java/org/iplantc/de/apps/client/gin/AppsGinModule.java

```diff
@@ -60,65 +65,69 @@
 
     @Override
     protected void configure() {
-        bind(new TypeLiteral<TreeStore<AppCategory>>() {})
-            .toProvider(AppCategoryTreeStoreProvider.class);
-        bind(new TypeLiteral<ListStore<App>>() {})
-            .toProvider(AppListStoreProvider.class);
-        bind(new TypeLiteral<TreeStore<OntologyHierarchy>>() {})
-                .toProvider(OntologyHierarchyTreeStoreProvider.class);
+        bind(new TypeLiteral<TreeStore<AppCategory>>() {
+        }).toProvider(AppCategoryTreeStoreProvider.class);
+        bind(new TypeLiteral<ListStore<App>>() {
+        }).toProvider(AppListStoreProvider.class);
+        bind(new TypeLiteral<TreeStore<OntologyHierarchy>>() {
+        }).toProvider(OntologyHierarchyTreeStoreProvider.class);
 
         bind(SubmitAppForPublicUseView.class).to(SubmitAppForPublicUseViewImpl.class);
         bind(SubmitAppForPublicUseView.Presenter.class).to(SubmitAppForPublicPresenter.class);
         bind(AppMetadataServiceFacade.class).to(AppMetadataServiceFacadeImpl.class);
         bind(OntologyServiceFacade.class).to(OntologyServiceFacadeImpl.class);
 
         // Main View
-        install(new GinFactoryModuleBuilder()
-                    .implement(AppsView.class, AppsViewImpl.class)
-                    .build(AppsViewFactory.class));
+        install(new GinFactoryModuleBuilder().implement(AppsView.class, AppsViewImpl.class)
```

Comment:
I keep meaning to find that setting I was talking about where IntelliJ will only format lines you've changed... 😖